src/nimlsp.nim

```diff
@@ -481,6 +481,7 @@ proc main(){.async.} =
               ).JsonNode
           else:
             debugEcho "Unknown request"
+            await message.error(errorCode = -32600,message="Unknown request:" & frame ,data = newJObject())
```

Comment:
Please use consistent spacing around commas and equal signs, this line should be `await message.error(errorCode = -32600, message = "Unknown request:" & frame, data = newJObject())`